test/bottomVisible/bottomVisible.test.js

```diff
@@ -1,12 +1,29 @@
 const test = require('tape');
-const bottomVisible = require('./bottomVisible.js');
+// const bottomVisible = require('./bottomVisible.js');
+
+// Make the fake properties used by the function
+const document = {
+  documentElement: {
+    clientHeight: 1080,
+    scrollHeight: 1080
+  }
+};
+
+const window = {
+  scrollY: 2160
+}
+
+// Override the function to use these fake properties
+const bottomVisible = () =>
+document.documentElement.clientHeight + window.scrollY >=
+(document.documentElement.scrollHeight || document.documentElement.clientHeight);
```

Comment:
Either use `jsdom` or revert these to their original. We currently have some trouble with the browser-specific snippets, so it'd be better to just revert changes to this file until we figure it all out.